pants.ini

```diff
@@ -220,6 +220,13 @@ no_warning_args: [
     '-S-nowarn',
   ]
 
+compiler_option_sets_enabled_args: {
+    # Needed to make scoverage CodeGrid highlighting work
+    'scoverage': [
```

Comment:
Rather than having to manually include this, we should automatically include it when `scoverage` is enabled. Probably somewhere around here? https://github.com/pantsbuild/pants/blob/4435b8ea14cc753d21f0c22377140cab2156c593/src/python/pants/backend/jvm/tasks/jvm_compile/zinc/zinc_compile.py#L324-L325